.travis.yml

```diff
@@ -45,46 +45,84 @@ jobs:
 
     - stage: test
       env:
-        - PROJECT=Firestore PLATFORM=iOS METHOD=xcodebuild
+        - PROJECT=Firebase PLATFORM=macOS
```

Comment:
The order in this file matters:
  * We try to get every sub-project a single top-level build on iOS as soon as possible.
  * We specifically don't want to have to wait for community-supported platforms for tests on primary platforms

Therefore I propose the following ordering:
  * The first three should be Firebase, Firestore, Functions for iOS
  * Then Firestore in cmake since that's a fully-supported alternate platform
  * Then other platforms
  * Then extra checks (like the sanitizers)

Reply:
Thanks for explaining. I presumed order doesn't matter (they're supposed to be run in parallel).

Reply:
They're started in order as capacity becomes available. When the system has zero load it's possible for up to 10 stages start in parallel (after `checks`) is done.

Feel free to add a comment up at the top to help future maintainers.